tests/xml.js

```diff
@@ -298,5 +306,17 @@ define([
         it("should round-trip newline", function () {
             eq(xml.query("\n").toString(), "\n");
         });
+
+        it("should not insert text inside output refs", function () {
```

Comment:
Can we at least add a comment that says something to the effect of
```suggestion
        it("should not insert text inside output refs", function () {
            // this test is verifying a quirk of jQuery 3.5.0 that results in
            // invalid XML transformations. It is not necessary to preserve
            // the behavior if/when jQuery no longer mangles XML this way.
```